app/javascript/src/components/Dashboard/Contacts/Pane/Create.jsx

```diff
@@ -0,0 +1,23 @@
+import React from "react";
+
+import { Pane, Typography } from "neetoui";
+
+import Form from "./Form";
+
+import { CONTACT_FORM_INITIAL_FORM_VALUES } from "../constants";
+
+const NewContactPane = ({ showPane, setShowPane }) => {
```

Comment:
Same review as my last review comment in this PR.

Reply:
I will rectify it.